docs/src/implicit/PDIRK.md

```diff
@@ -0,0 +1,12 @@
+# OrdinaryDiffEqPDIRK
+
+This article is a stub.
```

Comment:
@ChrisRackauckas write a sentence what PDIRK is about.

Reply:
PDIRK methods are parallel DIRK methods. SDIRK methods, or singly-diagonally implicit methods, have to build and solve a factorize a Jacobian of the form W = I-gamma*J where gamma is dependent on the chosen method. PDIRK methods use multiple different choices of gamma, i.e. W_i = I-gamma_i*J, which are all used in the update process. There are some advantages to this, as no SDIRK method can be a higher order than 5 while DIRK methods generally can have arbitrarily high order and lower error coefficients, leading to lower errors at larger `dt` sizes. With the right construction of the tableau, these matrices can be factorized and the underlying steps can be computed in parallel, which is why these are the parallel DIRK methods.

Note that this set is currently experimental as there are no parallel DIRK tableaus that achieve good performance in the literature.

Reply:
If there are no good DIRK tableaus, why is Kvaerno5 one of the recommended solvers?
https://docs.sciml.ai/DiffEqDocs/stable/solvers/ode_solve/#Stiff-Problems

Reply:
It's an (E)SDIRK tableau.